### PR TITLE
Add docs for pinning notebooks/collection to featured

### PR DESCRIPTION
**Pull request by tanya1108**
### Summary:
Docs for pinning `notebooks/collection`  to featured has been added.

**Screenshot**
FeaturedDocs

_**UPDATED PREVIEW LINK**_
[]https://jovian-codebase.readthedocs.io/en/tanya-make-docs-for-pinning-notebooks-collection-featured/user-guide/featured.html
**Queries**
- What other point needs to be added?

- What modifications are needed in the current change?

### CODE REVIEW

File: docs/user-guide/featured.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```eval_rst
+.. meta::
+   :description: Featured Notebooks and Collections.
+```
+
+<a href="https://jovian.ai?utm_source=docs" target=_blank> Jovian </a> provides you with the feature to highlight some of your Notebooks & Collections on profile overview. You can feature like those
+related to your project , or some of your favourite topics ,or frameworks/languages used, or other important qualities that you find useful on your profile.
+
+### How to add Notebooks/Collections in Featured Section?
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:11]: You have 4 numbered points under this heading, but actually they don't talk to `adding` except for the first point.

You can have heading for each points.

File: docs/user-guide/featured.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1. For each notebook and collection you have been provided with the option to Pin it.
+   Note that you cannot pin private/secret notebooks and private collections.
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:14]: `Sphinx-rtd-theme` (theme that we are using for our docs page) has support for this kind of section. 

You can add `Note` with this syntax
```
```eval_rst
.. note:: {text}
```
**Preview**
image

It been used in `verison.md` check it out for reference.

File: docs/user-guide/featured.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  <img src="https://i.imgur.com/oxzHbMc.png" class="screenshot" alt="Pin Notebook">
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:16]: You can use this https://imgur.com/ccxU4QF.png instead, the current screenshots has test notebooks with just one line in its preview image.

File: docs/user-guide/featured.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  <img src="https://i.imgur.com/5k71Moy.png" class="screenshot" alt="Pin Collection">
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:18]: Can you update the following screenshots with the same boundary as https://imgur.com/ccxU4QF.png to keep it consistent.

File: docs/user-guide/featured.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+2. Navigate to Overview section after you have pinned the notebook/collection.There you see
+   your featured Notebooks and Collections.
+   Similarly you can see the featured collection and notebooks for other users as well.
+
+   <img src="https://i.imgur.com/QhoQ2RT.jpg" class="screenshot" alt="OverView Tab">
+
+3. If you want to removal some notebook or collection from feature , simply unpin them
+   from Notebook/ Collection section.
+
+4. Note that you can pin maximum 6 collection/notebook at a time.
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:29]: This can be turned to a `Note` section too

File: docs/user-guide/featured.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+### How to add Notebooks/Collections in Featured Section?
+
+1. For each notebook and collection you have been provided with the option to Pin it.
+   Note that you cannot pin private/secret notebooks and private collections.
+
+  <img src="https://i.imgur.com/oxzHbMc.png" class="screenshot" alt="Pin Notebook">
+
+  <img src="https://i.imgur.com/5k71Moy.png" class="screenshot" alt="Pin Collection">
+
+2. Navigate to Overview section after you have pinned the notebook/collection.There you see
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:20]: Add a heading `View featured Notebooks/Collections` to this.

File: docs/user-guide/featured.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+   your featured Notebooks and Collections.
+   Similarly you can see the featured collection and notebooks for other users as well.
+
+   <img src="https://i.imgur.com/QhoQ2RT.jpg" class="screenshot" alt="OverView Tab">
+
+3. If you want to removal some notebook or collection from feature , simply unpin them
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:26]: Add a heading and screenshot to this as well.

File: docs/user-guide/featured.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:1]: Any time you create a new markdown file you should add it to `toctree` in `index.rst` to make it easy for users to navigate.
You can observe that the same order is been used to render the left pane in our docs page.
image

You can add `user-guide/featured` after `user-guide/topics`

File: docs/user-guide/featured.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+1. For each notebook and collection you have been provided with the option to Pin it.
+   Note that you cannot pin private/secret notebooks and private collections.
+
+  <img src="https://i.imgur.com/oxzHbMc.png" class="screenshot" alt="Pin Notebook">
+
+  <img src="https://i.imgur.com/5k71Moy.png" class="screenshot" alt="Pin Collection">
+
+2. Navigate to Overview section after you have pinned the notebook/collection.There you see
+   your featured Notebooks and Collections.
+   Similarly you can see the featured collection and notebooks for other users as well.
```
Comment (PrajwalPrashanth) [inline docs/user-guide/featured.md:22]: Notebook/Collection is different in each instance like
- collection and notebook
- notebook & collection
- notebook or collection

Please go over this PR and make it consistent

**Review by PrajwalPrashanth — CHANGES_REQUESTED** [review]

File: docs/user-guide/featured.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
+
+```eval_rst
+.. meta::
+   :description: Featured Notebooks and Collections.
+```
+
+<a href="https://jovian.ai?utm_source=docs" target=_blank> Jovian </a> provides you with the feature to highlight some of your Notebooks & Collections on profile overview. You can feature like those
+related to your project , or some of your favourite topics ,or frameworks/languages used, or other important qualities that you find useful on your profile.
```
[the remark this answers is not in the document]
Comment (tanya1108) [reply docs/user-guide/featured.md]: Added

File: docs/user-guide/featured.md (hunk 5 above)
Comment (tanya1108) [reply docs/user-guide/featured.md]: Added.

File: docs/user-guide/featured.md (hunk 7 above)
Comment (tanya1108) [reply docs/user-guide/featured.md]: New heading and screenshots added.

File: docs/user-guide/featured.md (hunk 9 above)
Comment (tanya1108) [reply docs/user-guide/featured.md]: Made the same instance everywhere.

**Comment by aakashns** [comment]
@PrajwalPrashanth please review

**Review by PrajwalPrashanth — APPROVED** [review]

**Pull request merged**